src/uu/tr/src/operation.rs

```diff
@@ -89,6 +90,10 @@ impl Display for BadSequence {
                     end_or_start_to_string(end)
                 )
             }
+            BadSequence::MultipleCharInEquivalence(s) => write!(
```

Comment:
To be consistent with the other match arms:
```suggestion
            Self::MultipleCharInEquivalence(s) => write!(
```